core/src/test/java/org/elasticsearch/search/suggest/completion/FuzzyOptionsTests.java

```diff
@@ -41,49 +47,54 @@ public static FuzzyOptions randomFuzzyOptions() {
-        builder.setFuzziness(original.getEditDistance())
-            .setFuzzyPrefixLength(original.getFuzzyPrefixLength())
-            .setFuzzyMinLength(original.getFuzzyMinLength())
-            .setMaxDeterminizedStates(original.getMaxDeterminizedStates())
-            .setTranspositions(original.isTranspositions())
-            .setUnicodeAware(original.isUnicodeAware());
+        builder.setFuzziness(original.getEditDistance()).setFuzzyPrefixLength(original.getFuzzyPrefixLength())
+                .setFuzzyMinLength(original.getFuzzyMinLength()).setMaxDeterminizedStates(original.getMaxDeterminizedStates())
+                .setTranspositions(original.isTranspositions()).setUnicodeAware(original.isUnicodeAware());
         switch (randomIntBetween(0, 5)) {
-            case 0:
-                builder.setFuzziness(randomValueOtherThan(original.getEditDistance(), () -> randomFrom(0, 1, 2)));
-                break;
-            case 1:
-                builder.setFuzzyPrefixLength(randomValueOtherThan(original.getFuzzyPrefixLength(), () ->
-                    randomIntBetween(1, 3)));
-                break;
-            case 2:
-                builder.setFuzzyMinLength(randomValueOtherThan(original.getFuzzyMinLength(), () ->
-                    randomIntBetween(1, 3)));
-                break;
-            case 3:
-                builder.setMaxDeterminizedStates(randomValueOtherThan(original.getMaxDeterminizedStates(), () ->
-                    randomIntBetween(1, 10)));
-                break;
-            case 4:
-                builder.setTranspositions(!original.isTranspositions());
-                break;
-            case 5:
-                builder.setUnicodeAware(!original.isUnicodeAware());
-                break;
+        case 0:
+            builder.setFuzziness(randomValueOtherThan(original.getEditDistance(), () -> randomFrom(0, 1, 2)));
+            break;
+        case 1:
+            builder.setFuzzyPrefixLength(randomValueOtherThan(original.getFuzzyPrefixLength(), () -> randomIntBetween(1, 3)));
+            break;
+        case 2:
+            builder.setFuzzyMinLength(randomValueOtherThan(original.getFuzzyMinLength(), () -> randomIntBetween(1, 3)));
+            break;
+        case 3:
+            builder.setMaxDeterminizedStates(randomValueOtherThan(original.getMaxDeterminizedStates(), () -> randomIntBetween(1, 10)));
+            break;
+        case 4:
+            builder.setTranspositions(!original.isTranspositions());
+            break;
+        case 5:
+            builder.setUnicodeAware(!original.isUnicodeAware());
+            break;
         }
         return builder.build();
     }
 
-    @Override
-    protected FuzzyOptions readFrom(StreamInput in) throws IOException {
-        return new FuzzyOptions(in);
+    /**
+     * Test serialization and deserialization
+     */
+    public void testSerialization() throws IOException {
+        for (int i = 0; i < NUMBER_OF_RUNS; i++) {
```

Comment:
I think we've been moving away from this sort of "poor man's repeat" and just doing the run once and hoping for CI to catch enough stuff. If we really aren't sure that is enough maybe we should add a couple of specific mutations to test every time?